packages/api/src/conversation/index.ts

```
@@ -321,11 +321,11 @@ export default class Conversation extends Ad4mModel {
     currentNewTopics = currentNewTopics.filter((topic) => topic.n && topic.n.trim() !== '');
 
     const topicMatches = await Topic.findAll(this.perspective, {
-      where: { topic: currentNewTopics.map((topic) => Literal.from(topic.n).toUrl()) },
+      where: { topic: currentNewTopics.map((topic) => topic.n) },
     });
     await Promise.all(
       currentNewTopics.map((topic) => {
```

Biome (2.4.4)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)